docs/ref/migration-operations.txt

```diff
@@ -477,6 +477,65 @@ For an example using ``SeparateDatabaseAndState``, see
 
 .. _writing-your-own-migration-operation:
 
+Operation category
+==================
+
```

Comment:
`.. versionadded:: 5.1` is missing. Also `.. _writing-your-own-migration-operation:` should be still attached to the `Writing your own` section.